setup.py

```diff
@@ -64,6 +65,7 @@
     "Flask-BabelEx>=0.9.2",
     "Flask-Breadcrumbs>=0.3.0",
     "Flask-Menu>=0.5.0",
+    "invenio-assets>=1.0.0a1",
```

Comment:
@lnielsen It is used in tests and in the example application. Should I remove it from the "install_requires"?